### ISSUE TITLE
Plugin fails when opening reitit (a leiningen monorepo)

### ISSUE THREAD

**Issue by ikitommi** [brcosta/clj-extras-plugin#17]
Hi, opening up reitit (https://github.com/metosin/reitit) project with Clojure Extras (0.5.4) enabled, I get the following error:

```clj
ava.lang.Throwable: Unexpected error. Please report an issue.

	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:182)
	at com.github.brcosta.cljstuffplugin.cljkondo.CljKondoProcessRunner$run$1$processHandler$1$1.onTextAvailable(CljKondoProcessRunner.kt:32)
	at jdk.internal.reflect.GeneratedMethodAccessor102.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.intellij.execution.process.ProcessHandler$2.invoke(ProcessHandler.java:247)
	at com.sun.proxy.$Proxy62.onTextAvailable(Unknown Source)
	at com.intellij.execution.process.ProcessHandler.notifyTextAvailable(ProcessHandler.java:221)
	at com.intellij.execution.process.ColoredProcessHandler.coloredTextAvailable(ColoredProcessHandler.java:68)
	at com.intellij.execution.process.AnsiEscapeDecoder.processTextChunk(AnsiEscapeDecoder.java:100)
	at com.intellij.execution.process.AnsiEscapeDecoder.escapeText(AnsiEscapeDecoder.java:64)
	at com.intellij.execution.process.ColoredProcessHandler.notifyTextAvailable(ColoredProcessHandler.java:57)
	at com.intellij.execution.process.BaseOSProcessHandler$SimpleOutputReader.onTextAvailable(BaseOSProcessHandler.java:155)
	at com.intellij.util.io.BaseOutputReader.sendText(BaseOutputReader.java:205)
	at com.intellij.util.io.BaseOutputReader.processInput(BaseOutputReader.java:189)
	at com.intellij.util.io.BaseOutputReader.readAvailableNonBlocking(BaseOutputReader.java:101)
	at com.intellij.util.io.BaseDataReader.readAvailable(BaseDataReader.java:74)
	at com.intellij.util.io.BaseDataReader.doRun(BaseDataReader.java:155)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:213)
	at com.intellij.util.io.BaseDataReader.lambda$start$0(BaseDataReader.java:50)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

disabling the plugin removes the error.

**Comment by brcosta** [comment]
I can reproduce it using clj-kondo binary option, can you check if you have the same problem using the built in clj-kondo? I'll also give it a try again later today.

**Comment by brcosta** [comment]
In any case I'll investigate what is happening when using the binary version later, thanks for reporting it :)

**Comment by brcosta** [comment]
Fixed on master (soon to be 0.6.1) :)
